contributors/devel/pull-requests.md

```diff
@@ -94,7 +100,7 @@ Release notes apply to PRs on the master branch. For cherry-pick PRs, see the [c
 **Comments**
 
 - If the PR does not need to be mentioned at release time, leave the `/release-note-none` comment command.
-- To create a release note, leave the `/release-note` comment command. This will generate a release note using the PR title by default OR the release-note block in the PR template, if it's filled in.
+- (_DEPRECATED_) To create a release note, leave the `/release-note` comment command. This will generate a release note using the PR title by default OR the release-note block in the PR template, if it's filled in.
```

Comment:
If this is deprecated, can we just remove it?

Reply:
I vote for removing it. If someone does try to use that command or the `/release-note-action-required` command, a comment warning that the command is deprecated is created.